src/mobu/cachemachine.py

```diff
@@ -30,8 +29,11 @@ def __init__(
         self._session = session
         self._token = token
         self._username = username
-        self._url = urljoin(
-            config.environment_url, "cachemachine/jupyter/available"
+        # Should be configurable between available and desired
+        self._url = (
+            config.environment_url
+            + "/cachemachine/jupyter/"
```

Comment:
Oh, I see, you got rid of `urljoin` instead. I'd rather use `urljoin` to assemble URLs, though.

Reply:
I'm not using urljoin anymore, so it won't.  There's no point in urljoin since we have slashes inside the component anyway: string concatenation is just fine.  If I use urljoin with two substitutions, then I have to nest the urljoins, and it just seems silly.